erpnext/accounts/report/budget_variance_report/budget_variance_report.js

```diff
@@ -75,7 +75,7 @@ frappe.query_reports["Budget Variance Report"] = {
 	"formatter": function (value, row, column, data, default_formatter) {
 		value = default_formatter(value, row, column, data);
 
-		if (column.fieldname.includes('variance')) {
+		if (column.fieldname.includes(__("variance"))) {
```

Comment:
@ernestoruiz89 This is not a user-facing string. Its an internal field name, it should not be translated

Reply:
If it is not translated it does not work, because the field is translated from python

![image](https://user-images.githubusercontent.com/1592496/199022041-355e1550-7647-45bc-82d4-319454e9222b.png)